pyaptly/main.py

```diff
@@ -22,13 +22,42 @@
 lg = logging.getLogger(__name__)
 
 
+def setup_logger(args):
+    """Setup the logger."""
+    global _logging_setup
+    root = logging.getLogger()
+    formatter = custom_logger.CustomFormatter()
+    if not _logging_setup:  # noqa
+        handler = logging.StreamHandler(sys.stderr)
+        handler.setFormatter(formatter)
+        root.addHandler(handler)
+        root.setLevel(logging.WARNING)
+        handler.setLevel(logging.WARNING)
+        if args.info:
+            root.setLevel(logging.INFO)
+            handler.setLevel(logging.INFO)
+        if args.debug:
+            root.setLevel(logging.DEBUG)
+            handler.setLevel(logging.DEBUG)
+        _logging_setup = True  # noqa
+
+
+def prepare(args):
+    """Set pretend mode, read config and load state."""
+    command.Command.pretend_mode = args.pretend
+
+    with open(args.config, "rb") as f:
+        cfg = tomli.load(f)
+    state_reader.state.read()
```

Comment:
Isn't this now `state_reader.get_state_reader().read()` as of #90 ?

Reply:
We will get a conflict for that.